File: spoken-words-recognition-api/Utils/Model/RawRecording.cs
using System;
using Utils.Interfaces;

namespace Utils.Model
{
    public class RawRecording : ITableRow
    {
        public Guid Id { get; set; }
        public DateTimeOffset Timestamp { get; set; }

        public Guid SpeakerId { get; set; }
        public string Accent { get; set; }
        public string Localization { get; set; }
        public Modification Modification { get; set; }

        public WordOccurence[] Words { get; set; }
    }
}

File: spoken-words-recognition-spa/src/app/services/api-client/RawRecording.ts
import { WordOccurence } from './WordOccurence';

export class RawRecording {

  id: string;
  speakerId: string;
  accent: string;
  localization: string;
  modification: string;
  words: WordOccurence[];

  public static prepare(raw: RawRecording) {
    raw.words.sort((x, y) => x.word > y.word ? 1 : -1);
  }
}
